frontend/public/kubevirt/components/nic.jsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import * as _ from 'lodash';
+import { Cog } from './utils/okdutils';
+import { List, ColHead, ListHeader, ResourceRow } from './factory/okdfactory';
+import { DASHES, BUS_VIRTIO, NETWORK_TYPE_MULTUS, NETWORK_TYPE_POD, NETWORK } from './utils/constants';
+import { deleteDeviceModal } from './modals/delete-device-modal';
+
+const getNetworkType = network => {
+  if (network.hasOwnProperty('pod')){
+    return NETWORK_TYPE_POD;
+  } else if (network.hasOwnProperty('multus')){
+    return NETWORK_TYPE_MULTUS;
+  }
+  return DASHES;
+};
+
+const menuActionDelete = (vm, nic) => ({
+  label: 'Delete',
+  callback: () => deleteDeviceModal({
+    type: NETWORK,
+    device: nic,
+    vm: vm
+  })
+});
+
+const getActions = (vm, nic) => {
+  const actions = [menuActionDelete];
+  return actions.map(a => a(vm, nic));
+};
+
+
+const NicHeader = props => <ListHeader>
+  <ColHead {...props} className="col-lg-3 col-md-3 col-sm-3 col-xs-4" sortField="name">Nic Name</ColHead>
```

Comment:
quite a lot of copy&paste.
Since this is aligned with similar OKD core code, I do not insist on refactoring that, just if you don't like it either :-)

Reply:
className was moved to `const` which is reused in other places